articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md

```diff
@@ -244,6 +239,9 @@ Once you have completed the movement of the data, it is a good idea to refresh t
 
 ```sql
 UPDATE STATISTICS [dbo].[FactInternetSales];
+DROP TABLE [dbo].[FactInternetSales_20000101]
+DROP TABLE [dbo].[FactInternetSales_20000101_20010101]
+
 ```
 
```

Comment:
```suggestion
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`. Alternatively, you may want to keep empty tables for regular, automated partition switches.
```

Reply:
@lucaferrari77 this PR isn't ready, I'm not sure what happened but please unresolve this comment and commit the suggestion "Finally, in the case..."

Reply:
@WilliamDAssafMSFT, I cannot "Unresolve" this. It only shows "Resolve conversation"
Not sure how to proceed.
![image](https://user-images.githubusercontent.com/90248940/172113058-c739529b-239f-4610-a299-06bd6a97790e.png)

Reply:
Just above that - I can see the two buttons in your screenshot - do you have an option to **Commit Suggestion**?

Reply:
If not, @Lucaferrari77 can you please add the following text as a new commit?

```
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`. Alternatively, you may want to keep empty tables for regular, automated partition switches.
```